app/views/events/index.html.haml

```diff
@@ -19,10 +19,14 @@
           = link_to "", event, class: "pin", data: { latitude: event.latitude, longitude: event.longitude }
     .col-sm-12.col-md-6
       = render @upcoming_events
-
 - unless @past_events.empty?
   %hr
-  %h4 Past 24 Pull Requests Events
   .row
+    %p
+      %button.btn.btn-default.toggle-past-events
+        %span Show
+        Past 24 Pull Requests Events
```

Comment:
Yes, just let me understand before push.
Here we have a button to show/hide and the h4. Your suggestion is remove the button to avoid repetition and add the handle click on h4?